src/components/ScholarshipAmountField.jsx

```diff
@@ -37,65 +41,76 @@ function ScholarshipAmountField(props) {
-        onChange={(e) => {
-          const val = parseInt(e.target.value, 10);
-          updateAmount(minAmount, val || 0);
-        }}
-        disabled={!isRange}
-      />
-    </div>
+    <Grid container alignItems="center">
+      <Grid item>
+        <AmountTextField
+          error={isRange && error && (!!maxAmount || minAmount >= maxAmount)}
+          value={minAmount || ''}
+          onChange={(e) => {
+            const val = parseInt(e.target.value, 10);
+            updateAmount(val || 0, maxAmount);
+          }}
+          disabled={!isRange}
+        />
+      </Grid>
+
+      <Grid item>
+        <RemoveIcon className={classes.dash} />
+      </Grid>
+
+      <Grid item>
+        <AmountTextField
+          error={isRange && error && !minAmount && !maxAmount}
+          value={maxAmount || ''}
+          onChange={(e) => {
+            const val = parseInt(e.target.value, 10);
+            updateAmount(minAmount, val || 0);
+          }}
+          disabled={!isRange}
+        />
+      </Grid>
+    </Grid>
   );
   labels[AmountType.Fixed] = (
-    <div className={classes.amountFieldStyle}>
-      Fixed Amount:
-      <AmountTextField
-        error={isFixed && error}
-        value={minAmount || ''}
-        onChange={(e) => {
-          const val = parseInt(e.target.value, 10);
-          updateAmount(val || 0, val || 0);
-        }}
-        disabled={!isFixed}
-      />
-    </div>
+    <AmountTextField
+      error={isFixed && error}
+      value={minAmount || ''}
+      onChange={(e) => {
+        const val = parseInt(e.target.value, 10);
+        updateAmount(val || 0, val || 0);
+      }}
+      disabled={!isFixed}
+    />
   );
-  labels[AmountType.FullTuition] = 'Full Tuition';
+  labels[AmountType.FullTuition] = <AmountTextField disabled />;
 
   return (
-    <FormControl error={error} className={className}>
-      <FormLabel>Amount Type *</FormLabel>
-      <RadioGroup value={amountType} onChange={onTypeChange}>
-        {Object.values(AmountType).map((type) => (
-          <FormControlLabel
-            key={type}
-            value={type}
-            control={<Radio />}
-            label={labels[type] || 'Unknown'}
-          />
-        ))}
-      </RadioGroup>
+    <>
+      <InputLabel className={classes.inputLabel}>Award Amount *</InputLabel>
+      <Grid container spacing={3}>
+        <Grid item>
+          <Select
+            className={classes.amountSelect}
+            variant="outlined"
+            value={amountType}
+            onChange={onTypeChange}
+            displayEmpty>
+            <MenuItem value={null}>None</MenuItem>
+            <MenuItem value={AmountType.Unknown}>Unknown</MenuItem>
+            <MenuItem value={AmountType.Fixed}>Fixed</MenuItem>
+            <MenuItem value={AmountType.Range}>Range</MenuItem>
+            <MenuItem value={AmountType.FullTuition}>Full Tuition</MenuItem>
+          </Select>
+        </Grid>
+
+        <Grid item>{labels[amountType]}</Grid>
```

Comment:
This'll fail if `amountType` isn't specified (it defaults to `null` at the moment)